docs/DeveloperGuide.adoc

```diff
@@ -137,76 +137,182 @@ image:BetterModelClassDiagram.png[]
+=== Undo/Redo feature
+==== Implementation
 
-The undo/redo mechanism is facilitated by `VersionedAddressBook`.
-It extends `AddressBook` with an undo/redo history, stored internally as an `addressBookStateList` and `currentStatePointer`.
+The undo/redo mechanism is facilitated by `VersionedXpire`.
+It extends `Xpire` with an undo/redo history, stored internally as an `XpireStateList` and `currentStatePointer`.
 Additionally, it implements the following operations:
 
-* `VersionedAddressBook#commit()` -- Saves the current address book state in its history.
-* `VersionedAddressBook#undo()` -- Restores the previous address book state from its history.
-* `VersionedAddressBook#redo()` -- Restores a previously undone address book state from its history.
+* `VersionedXpire#commit()` -- Saves the current Xpire state in its history.
+* `VersionedXpire#undo()` -- Restores the previous Xpire state from its history.
+* `VersionedXpire#redo()` -- Restores a previously undone Xpire state from its history.
 
-These operations are exposed in the `Model` interface as `Model#commitAddressBook()`, `Model#undoAddressBook()` and `Model#redoAddressBook()` respectively.
+These operations are exposed in the `Model` interface as `Model#commitXpire()`, `Model#undoXpire()` and `Model#redoXpire()` respectively.
 
 Given below is an example usage scenario and how the undo/redo mechanism behaves at each step.
 
-Step 1. The user launches the application for the first time. The `VersionedAddressBook` will be initialized with the initial address book state, and the `currentStatePointer` pointing to that single address book state.
+Step 1. The user launches the application for the first time. The `VersionedXpire` will be initialized with the initial Xpire state, and the `currentStatePointer` pointing to that single Xpire state.
 
 image::UndoRedoState0.png[]
 
-Step 2. The user executes `delete 5` command to delete the 5th person in the address book. The `delete` command calls `Model#commitAddressBook()`, causing the modified state of the address book after the `delete 5` command executes to be saved in the `addressBookStateList`, and the `currentStatePointer` is shifted to the newly inserted address book state.
+Step 2. The user executes `delete 5` command to delete the 5th item in the Xpire. The `delete` command calls `Model#commitXpire()`, causing the modified state of the Xpire after the `delete 5` command executes to be saved in the `XpireStateList`, and the `currentStatePointer` is shifted to the newly inserted Xpire state.
 
 image::UndoRedoState1.png[]
 
-Step 3. The user executes `add n/David ...` to add a new person. The `add` command also calls `Model#commitAddressBook()`, causing another modified address book state to be saved into the `addressBookStateList`.
+Step 3. The user executes `add n/David ...` to add a new item. The `add` command also calls `Model#commitXpire()`, causing another modified Xpire state to be saved into the `XpireStateList`.
```

Comment:
Oops, I think you forgot to update the add format. Perhaps you could change the "add n/David" to "add|Strawberry|3/2/2020|1"!

Reply:
Noted, will make the changes :)